<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyntroGloveViewClass</class>
 <widget class="QMainWindow" name="SyntroGloveViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyntroGloveView</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionBasicSetup"/>
    <addaction name="actionSelectSource"/>
    <addaction name="actionSelectFusionAlgorithm"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionSelectSource"/>
   <addaction name="separator"/>
   <addaction name="actionSelectFusionAlgorithm"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="onStats">
   <property name="text">
    <string>Display stats</string>
   </property>
  </action>
  <action name="actionShow_date">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show date</string>
   </property>
  </action>
  <action name="actionShow_time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show time</string>
   </property>
  </action>
  <action name="actionText_color">
   <property name="text">
    <string>Text color ...</string>
   </property>
  </action>
  <action name="actionShow_name">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show name</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBasicSetup">
   <property name="text">
    <string>Basic setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelectSource_2">
   <property name="text">
    <string>Video streams ...</string>
   </property>
   <property name="iconText">
    <string>Video Streams</string>
   </property>
  </action>
  <action name="actionAudioSetup">
   <property name="text">
    <string>Audio setup</string>
   </property>
   <property name="iconText">
    <string>Audio Setup</string>
   </property>
  </action>
  <action name="actionSelectSource">
   <property name="text">
    <string>Select source</string>
   </property>
  </action>
  <action name="actionSelectFusionAlgorithm">
   <property name="text">
    <string>Select fusion algorithm</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
